### DISCUSSION TITLE
Harvard CS50's Introduction to AI with Python - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#312] [Announcements]
# Harvard CS50's Introduction to AI with Python - CS自学指南

CS自学指南

https://csdiy.wiki/%E4%BA%BA%E5%B7%A5%E6%99%BA%E8%83%BD/CS50/

**Comment by Lipeng-Lai** [comment]
我创建了一个学习交流 *CS50* 的群 ： 738825142， 大家感兴趣的可以进群交流学习呀~

**Reply by Yuxin-Qiao** [reply]
搜索不到群号

**Reply by Mamihla** [reply]
群好像解散了吧

**Comment by wuyafei** [comment]
推荐一个翻译非常不错的中英字幕版本：https://www.bilibili.com/video/BV1pk4y137dN/?vd_source=2b9f6fcc0b53b5f5e1607a9e3e8c83bd

**Reply by yzzyzyyz** [reply]
不知道“非常不错”在哪儿，建议新同学换其他版本，推荐带“精翻”字样的

**Reply by carterPOI** [reply]
没找到有精翻字样的视频.在bili看到一个用豆包翻译的... 还有youtu自带的字幕和用ai翻译的..

**Reply by carterPOI** [reply]
感觉是不是有官方字幕啊 看youtu的就行? 因为我看cs50官方有一个video player,都可以定位到每一条字幕的

**Comment by dyccyber** [comment]
这个官网上的submmit到对应分支下能test代码吗

**Comment by AnsonJIang-Pku** [comment]
这门课如何测试自己代码的正确性呢？

**Reply by KnDellx956669** [reply]
我把作业submit了，但是progress里面也看不见反应

**Comment by gogo-100** [comment]


**Reply by gogo-100** [reply]
2024版有自动测试了🥰

**Comment by C202044zxy** [comment]
traffic竟然没有标算 /kk

**Comment by Layoomiety-why** [comment]
为什么我找不到这门课的作业

**Comment by dghtucs** [comment]
建了一个课程QQ 群，可入763420096

**Comment by 1nram1** [comment]
flyingPig部分project没写，24版project最后一个做了更新，发一个我自己的实现
https://github.com/1nram1/CS50-Al

**Comment by jjking9** [comment]
有人现在在学吗 一起交流一下

**Reply by hiddencapule** [reply]
dd

**Reply by jaisonZheng** [reply]
dd

**Reply by hiddencapule** [reply]
hiddencapsulel@gmail.com

**Comment by JohnRothan** [comment]
1

**Comment by JohnRothan** [comment]
1

**Comment by hiddencapule** [comment]
留个邮箱兄弟
________________________________

**Comment by Autumn-Feng61** [comment]
有人知道作业提交到GitHub上吗，为什么我提交到GitHub，课程网站上显示我没提交

**Reply by jaisonZheng** [reply]
有没有上过cs50，提交方式是类似的，可以通过submit50提交，用git的话会稍微麻烦点
